tests/framework/utils/k8s_helper.go

```diff
@@ -227,6 +227,9 @@ func (k8sh *K8sHelper) ExecToolboxWithRetry(retries int, namespace, command stri
 // ResourceOperation performs a kubectl action on a pod definition
 func (k8sh *K8sHelper) ResourceOperation(action string, manifest string) error {
 	args := []string{action, "-f", "-"}
+	if action == "apply" {
+		args = []string{"apply", "--server-side=true", "--force-conflicts", "-f", "-"}
```

Comment:
In our upgrade guide, will we need to document this option for applying the crds?

Reply:
Yes, I think so. Working to test that now.

Reply:
I didn't have to do this in my testing. Maybe it was only an issue when there were the other things not happening quite right? We'll make sure that removing this from the helper.go still works.

Reply:
We seem to need this in CI, but I didn't in testing. In CI we use `create` to create resources rather than `apply`, so that may be it.

Reply:
I went ahead and changed the upgrade doc to reflect `--server-side`